lib/node_modules/@stdlib/math/base/special/croundn/README.md

```diff
@@ -95,38 +116,140 @@ var bool = ( v === out );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+var uniform = require( '@stdlib/random/base/uniform' ).factory;
 var Complex128 = require( '@stdlib/complex/float64' );
-var randu = require( '@stdlib/random/base/randu' );
-var ceil = require( '@stdlib/math/base/special/ceil' );
-var real = require( '@stdlib/complex/real' );
-var imag = require( '@stdlib/complex/imag' );
+var floor = require( '@stdlib/math/base/special/floor' );
 var croundn = require( '@stdlib/math/base/special/croundn' );
 
-var re;
-var im;
+var randn = uniform( -5.0, 0.0 );
+var rand = uniform( -50.0, 50.0 );
```

Comment:
```suggestion
var rand1 = uniform( -5.0, 0.0 );
var rand2 = uniform( -50.0, 50.0 );
```

`randn` is a common alias for a PRNG generating normally-distributed pseudorandom numbers. Renaming to avoid any potential confusion.